app/cdap/components/NamespaceAdmin/store/ActionCreator.ts

```diff
@@ -189,6 +197,111 @@ export function deleteConnection(conn: IConnection) {
   });
 }
 
+export const getSourceControlManagement = (namespace) => {
```

Comment:
@njbriggs is this a better way to write compared to the previous version?

Reply:
In general, yes. One thing is I think `catchError` should come last where possible.